File: module_main/src/main/java/debug/LoadingActivity.java
package debug;

import android.content.Intent;
import android.os.Bundle;

import com.czm.module.common.base.BaseActivity;
import com.czm.module.common.utils.ToastUtils;
public class LoadingActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (!isTaskRoot()) {
        /* If this is not the root activity */
            Intent intent = getIntent();
            String action = intent.getAction();
            if (intent.hasCategory(Intent.CATEGORY_LAUNCHER) && Intent.ACTION_MAIN.equals(action)) {
                finish();
                return;
            }
        }
//        setContentView(R.layout.activity_loading);
//        StatusBarUtil.transparencyBar(this, null, false);
    }
    @Override
    protected void onResume() {
        super.onResume();
        ToastUtils.showLongToast("组件模式运行");
//        new Handler().postDelayed(new Runnable() {
//            @Override
//            public void run() {
//
//            }
//        }, 2000);
    }
}

File: module_kotlin/src/main/java/debug/LoadingActivity.kt
package debug

import android.content.Intent
import android.os.Bundle

import com.czm.module.common.base.BaseActivity
import com.czm.module.common.utils.ToastUtils

class LoadingActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (!isTaskRoot) {
            /* If this is not the root activity */
            val intent = intent
            val action = intent.action
            if (intent.hasCategory(Intent.CATEGORY_LAUNCHER) && Intent.ACTION_MAIN == action) {
                finish()
                return
            }
        }
        //        setContentView(R.layout.activity_loading);
        //        StatusBarUtil.transparencyBar(this, null, false);
    }

    override fun onResume() {
        super.onResume()
        ToastUtils.showLongToast("组件模式运行")
        //        new Handler().postDelayed(new Runnable() {
        //            @Override
        //            public void run() {
        //
        //            }
        //        }, 2000);
    }
}
